internal/runtime/runtime.go

```diff
@@ -346,6 +346,7 @@ func (rt *ResourceGroupRuntime) evaluateDynamicVariables() error {
 	// and are resolved after all the dependencies are resolved.
 
 	resolvedResources := maps.Keys(rt.resolvedResources)
+	resolvedResources = append(resolvedResources, "schema")
```

Comment:
looks we allowed this during the RG parsing layer, but we didn't during the runtime execution layer. Great catch!